lib/MusicBrainz/Server/Data/ArtistCredit.pm

```diff
@@ -195,14 +195,6 @@ sub find_for_artist {
     );
 }
 
-sub _clean {
```

Comment:
Done. (I also removed the `no Moose` at the end, because `namespace::autoclean` will remove the Moose imports already.)